src/content/docs/serverless-function-monitoring/azure-function-monitoring/env-variables-azure.mdx

```diff
@@ -8,7 +8,139 @@ tags:
   - Configuration
   - Secrets Management
 ---
-In addition to the [required and optional environment variables for the .NET agent](/docs/apm/agents/net-agent/configuration/net-agent-configuration/#environment-variables), the following environment variables specific to Azure function monitoring can be set:
+
+Environment variables are a way to store configuration and secrets outside of your code. The following tables display the mandatory variables and runtime specific variables that you can set to monitor your Lambda functions with New Relic.
+
+## Mandatory environment variables
+Regardless of your runtime, you must set the following environment variables to monitor your Lambda functions with New Relic:
+
+ <table>
+  <thead>
+    <tr>
+      <th style={{ width: "200px" }}>Variable Name</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>`NEW_RELIC_ACCOUNT_ID`</td>
+      <td>Specifies your New Relic account ID</td>
+    </tr>
+    <tr>
+      <td>`NEW_RELIC_LICENSE_KEY`</td>
+      <td>Specifies your New Relic ingest key, overriding Secrets Manager</td>
+    </tr>
+  </tbody>
+</table>
+
+
+## Additional environment variables
+
+Based on your runtime, you can set the following environment variables to further configure the New Relic Lambda monitoring:
+
+
+
+<Tabs>
+  <TabsBar>
+    <TabsBarItem id="nodejs">Node.js</TabsBarItem>
+    <TabsBarItem id="python">Python</TabsBarItem>
+    <TabsBarItem id="net">.NET</TabsBarItem>
+
+  </TabsBar>
+
+  <TabsPages>
+    <TabsPageItem id="nodejs">
+<table>
+    <thead>
+        <tr>
+            <th>Environment variable</th>
+            <th>Default value</th>
+            <th>Options</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>`NODE_OPTIONS`</td>
+            <td></td>
+            <td>`-r newrelic`</td>
+            <td>**Required**: Inject the agent before your Azure application functions are invoked. </td>
+        </tr>
+        <tr>
+            <td>`NEW_RELIC_APP_NAME`</td>
+            <td></td>
+            <td></td>
+            <td>Set the application name, though it is not used in the New Relic UI</td>
+        </tr>
+        <tr>
+            <td>`NEW_RELIC_DISTRIBUTED_TRACING_ENABLED`</td>
+            <td>`true`</td>
+            <td>`true`, `false`</td>
+            <td>Generate traces by enabling Distributed Tracing</td>
+        </tr>
```

Comment:
Since the Node Agent's implementation of Distributed Tracing is on by default, this environment variable is required only if customers are turning DT off. I wouldn't expect it to be foregrounded in this configuration documentation.